openreview/workflows/workflow_process/area_chairs_group_template_process.py

```diff
@@ -0,0 +1,57 @@
+def process(client, edit, invitation):
```

Comment:
this is something that I would like to refactor to be the same for any group that we want to create that requires messaging and recruitment.